src/PHPUnit/Controller/AbstractHttpControllerTestCase.php

```diff
@@ -56,7 +56,7 @@ public function assertResponseReasonPhrase($phrase)
     public function assertHasResponseHeader($header)
     {
         $responseHeader = $this->getResponseHeader($header);
-        if (false === $responseHeader) {
+        if (! $responseHeader) {
```

Comment:
Overall, many of these changes are stylistic, and they introduce regressions.

For example, what happens if a header is `''` here? `false === $responseHeader` is much more precise, and should be kept, in this and other header comparison checks.

Reply:
This feedback applies to similarly changed code locations